.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+## RVM Version
```

Comment:
Sometimes you make headers starting all words from capital letters (eg. RVM Version), while sometimes not (eg Expected behavior). I would keep it consistent and capitalize only first word always (RVM version, Operating system, etc)